config/config.go

```diff
@@ -134,7 +134,8 @@ type Config struct {
 
 	Auth *auth.Config
 
-	EvmStore *evm.EvmStoreConfig
+	EvmStore  *evm.EvmStoreConfig
+	EvmConfig *evm.EvmConfig
```

Comment:
Don't need a whole subsection, just add a top level setting named `AllowNamedEVMContracts` that defaults to `false`.